dev/Generated/NumberBox.properties.h

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// DO NOT EDIT! This file was generated by CustomTasks.DependencyPropertyCodeGen
+#pragma once
+
+class NumberBoxProperties
```

Comment:
Not sure what the standard is in this repo, but it always helps to have some summary comments before each class to at least specify its purpose

Reply:
This file is generated and checked in for debugging purposes so it's not generally supposed to be reviewed.